tests/python/unittest/test_gluon.py

```diff
@@ -336,6 +339,34 @@ def hybrid_forward(self, F, x):
     net.hybridize()
     assert isinstance(net(mx.nd.zeros((16, 10))), mx.nd.NDArray)
 
+    # Test case to verify if initializing the SymbolBlock from a model with params 
+    # other than fp32 param dtype.
+
+    # 1. Load a resnet model, cast it to fp64 and export
+    tmp = tempfile.mkdtemp()
```

Comment:
should we delete the temporary directory when done with it?

Reply:
Temp gets automatically cleaned up.

Reply:
According to python docs, it seems to be deleted by the user. [The user of mkdtemp() is responsible for deleting the temporary directory and its contents when done with it.](https://docs.python.org/2/library/tempfile.html#tempfile.mkdtemp)

Reply:
Right. Thanks. I meant, temp gets automatically cleaned up after all the tests (test session).

Reply:
@marcoabreu - Can you please confirm if my understanding is correct? If not, I will add code to delete the temp directory created in the tests. Also, I see similar behavior in all other tests, where it creates temp dir, but, assumes, it will be cleaned up the system.